packages/oncokb-frontend-commons/src/components/RefComponent.tsx

```diff
@@ -11,46 +11,64 @@ export default class RefComponent extends React.Component<{
     componentType: 'tooltip' | 'linkout';
 }> {
     render() {
-        const parts = this.props.content.split(/pmid|nct/i);
+        const parts = this.props.content.split(/pmid|nct|abstract/i);
 
         if (parts.length < 2) {
             return <span>{this.props.content}</span>;
         }
 
-        const ids = parts[1].match(/[0-9]+/g);
+        // Slicing from index 1 to end to rejoin the abstract title and link
```

Comment:
to be safe i would surround all of this parsing logic in try/catch just to avoid crashing in some difficult to forsee edgecase.   I would also extract this parsing/composition into a nicely named function and provide in the comment an example of input AND output.   this should probably have a unit test too.